ui/raidboss/data/06-ew/ultimate/the_omega_protocol.txt

```diff
@@ -207,15 +209,19 @@ hideall "--sync--"
 789.0 "Hello, Near World 3" #sync / 1[56]:[^:]*:Omega-M:7B8A:/
 
 # Sigma Version
-# TODO: Adjust targetable/untargetable
-791.0 "--targetable--"
+# TODO: Account for patterns
+791.1 "--targetable--"
+805.2 "Solar Ray 1" sync / 1[56]:[^:]*:Omega-M:81AC:/
+808.5 "Solar Ray 2" sync / 1[56]:[^:]*:Omega-M:7B01:/
 815.6 "--sync--" sync / 1[56]:[^:]*:Omega-M:7B42:/
 821.7 "Run: ****mi* (Sigma Version)" sync / 1[56]:[^:]*:Omega-M:8014:/
 824.7 "--untargetable--"
 
 839.0 "Subject Simulation F" sync / 1[56]:[^:]*:Omega-M:7F2F:/
+840.4 "Program Loop" sync / 1[56]:[^:]*:Omega:7B98:/
 840.0 "--sync--" sync / 1[56]:[^:]*:Omega-M:7B14:/
 841.1 "--sync--" sync / 1[56]:[^:]*:Omega-M:7B16:/
+844.0 "Wave Cannon" sync / 1[56]:[^:]*:Omega:7B73:/
 843.8 "Hyper Pulse" sync / 1[56]:[^:]*:Right Arm Unit:7B72:/
 843.9 "Wave Cannon" #sync / 1[56]:[^:]*:Omega:7B74:/
```

Comment:
Is this another self-casted / damage redundancy between 7B73 and 7B74? Can the damage one be ignored?

Reply:
Yea, 7B74 looks to be the damage one and 7B73 the self-cast.